src/main/java/bio/terra/service/filedata/DrsService.java

```diff
@@ -429,7 +429,7 @@ private DRSContentsObject makeDrsContentsObject(FSItem fsObject, String snapshot
       return Collections.emptyList();
     }
 
-    String hdr = String.format("Authorization: Bearer %s", authUser.getRequiredToken());
+    String hdr = String.format("Authorization: Bearer %s", authUser.getToken());
```

Comment:
I wonder if this might break the method here:
bio.terra.service.filedata.DrsService#makeAuthHeader

I'm not sure that we even want to support that case (where there's not token)

I just wonder if there is any fallout from always requiring a token

Reply:
That's a good point! The token is a string in the terra AuthenticatedUserRequest class instead of an optional, and throws an error if the token is null: https://github.com/DataBiosphere/terra-common-lib/blob/0.0.51/src/main/java/bio/terra/common/iam/AuthenticatedUserRequest.java#L80-L81